managed-databases/redis/reference-content/ensuring-data-persistence.mdx

```diff
@@ -0,0 +1,54 @@
+---
+meta:
+  title: Ensuring data persistence
+  description: Learn more about how to ensuring data persistence in Managed Database for Redis™.
+content:
+  h1: Ensuring data persistence
+  paragraph: Learn more about how to ensuring data persistence in Managed Database for Redis™.
+tags: databases user redis username password
+categories:
+  - managed-databases
+  - redis
+---
+
+Ensuring data persistence is ensuring that the data from your applications is saved, in memory or on a disk, even when there are no ongoing sessions where the data is treated. The method used to preserve data between sessions, and avoid data loss varies depending on the compute resource you use to treat your data. It also depends on the way your data is stored.
```

Comment:
Data persistence ensures that the data from your applications is saved in memory or on a disk...